src/views/AlumniUpdateForm/index.js

```diff
@@ -0,0 +1,21 @@
+import GordonLoader from 'components/Loader';
+import GordonUnauthorized from 'components/GordonUnauthorized';
+import { useUser } from 'hooks';
+import UpdateForm from './components/UpdateForm';
+
+/**
+ * A form for alumni to request an update to their profile information.
+ */
+
+const AlumniUpdateForm = (props) => {
+  const { profile, loading } = useUser();
+  return loading ? (
+    <GordonLoader />
+  ) : profile ? (
+    <UpdateForm profile={profile} authentication={props.authentication} />
+  ) : (
+    <GordonUnauthorized feature={'the Alumni Update Page'} />
+  );
```

Comment:
I think this is more readable as `if` statements than nested ternary expressions.
```suggestion
  if (loading) return <GordonLoader />
  
  if (!profile) return <GordonUnauthorized feature={'the Alumni Update Page'} />
  
  return <UpdateForm profile={profile} />
```